docs/v3/manage/self-host.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Host Prefect server
+title: Prefect server
```

Comment:
I think we want to keep "Host" for consistency with other navigation items/titles so that they all start with a verb. That was the suggestion from Stellar.

Reply:
I buy that. This commit puts 'Host' back in the page title. https://github.com/PrefectHQ/prefect/pull/16073/commits/cdbcb9efe2e1608791d59e5baf999897e8b05039

However, I propose we keep the sidebar title for this topic as just "Prefect server" to match the corresponding "Prefect Cloud" group. In the sidebar, the implied verb is "Operate" for both.

<img width="258" alt="image" src="https://github.com/user-attachments/assets/1e7f25de-0b3e-419a-9b1f-edf6a88b826d">